toolchain/sem_ir/inst.h

```diff
@@ -445,6 +453,29 @@ class InstBlockStore : public BlockValueStore<InstBlockId> {
   }
 };
 
+inline auto CarbonHashValue(const Inst& value, uint64_t seed) -> HashCode {
```

Comment:
Similar to above, I'd appreciate some comments explaining what's going on here.

Reply:
Added a cross-reference to the documentation in hashing.h.